src/main/java/com/amazon/ion/impl/bin/IonManagedWriter_1_1.kt

```diff
@@ -335,6 +344,12 @@ internal class IonManagedWriter_1_1(
 
     override fun isStreamCopyOptimized(): Boolean = false
 
+    override fun writeValues(reader: IonReader, symbolIdTransformer: IntTransformer) {
+        sidTransformer = symbolIdTransformer
+        writeValues(reader)
+        sidTransformer = null
```

Comment:
I know it's unlikely we're going to encounter a scenario where this matters, but if we do... it could be really perplexing if this is not getting reset.
```suggestion
        try {
            sidTransformer = symbolIdTransformer
            writeValues(reader)
        } finally {
            sidTransformer = null
        }
```

Reply:
Yep, good call.